query-engine/core/src/executor/mod.rs

```diff
@@ -131,3 +132,53 @@ pub trait TransactionManager {
 pub fn get_current_dispatcher() -> Dispatch {
     tracing::dispatcher::get_default(|current| current.clone())
 }
+
+#[cfg(not(target_arch = "wasm32"))]
+pub(crate) mod task {
+    use super::*;
+
+    pub type JoinHandle<T> = tokio::task::JoinHandle<T>;
+
+    pub fn spawn<T>(future: T) -> JoinHandle<T::Output>
+    where
+        T: Future + Send + 'static,
+        T::Output: Send + 'static,
+    {
+        tokio::spawn(future)
+    }
```

Comment:
This is the `task::spawn` function used by native targets, which relies on `tokio` under the hood.